src/storage-client/src/healthcheck.rs

```diff
@@ -152,3 +152,15 @@ pub static REPLICA_METRICS_HISTORY_DESC: LazyLock<RelationDesc> = LazyLock::new(
         )
         .finish()
 });
+
+pub static WALLCLOCK_LAG_HISTORY_DESC: LazyLock<RelationDesc> = LazyLock::new(|| {
+    RelationDesc::builder()
+        .with_column("object_id", ScalarType::String.nullable(false))
+        .with_column("replica_id", ScalarType::String.nullable(true))
+        .with_column("lag_ms", ScalarType::UInt64.nullable(false))
```

Comment:
I'm exposing the lag as a `UInt64`, rather than an `Interval`, to reduce the data usage of this relation. If we are too annoyed with compromising usability for resource efficiency, we could also store an `Interval` here and then only translate it to a `UInt64` in `mz_wallclock_global_lag_recent_history`, which is the view that gets indexed.

Reply:
Do we know that this could be problematic? To me it feels like premature optimization but I may be unaware of known perf results.

Reply:
(As mentioned on Slack, I also have a weak preference for starting with an `Interval` type here!)